news/2021-12-02-skinning-contest-tides-of-winter-accouncement.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: post
+title: "Skinning Contest: Tides of Winter"
+date: 2021-12-02 12:00:00 +0000
+---
+
+The Skinning Contest returns as Winter cracks the door open. Grab a cup of hot chocolate and tune in for all the details!
+
+![](/wiki/shared/news/2021-12-02-skinning-contest-tides-of-winter-accouncement/announcement.jpg)
+
+Months after the launch of our first iteration, we're once again here to call forth all the skinners for the second time to show off their talents in the Skinning Contest, hosted by one of the largest skinning communities, [skinship](https://skinship.xyz/). For those unfamiliar with what this is all about, check out the wiki page for the previous iteration [here](/wiki/Contests/Skinning_Contest/1).
+
+Since the Winter is here and the snowy days are nearing, we have decided to adopt a theme that will reflect the season properly: **Tides of Winter**! As you might have guessed, in this iteration of the contest, we expect submissions all themed around anything Winter-related. Whether it fills people joy with the everlasting Christmas spirit, or pulls out a different take on Winter sports, anything is acceptable as long as it fits the theme.
```

Comment:
```suggestion
Since winter is here and the snowy days are nearing, we have decided to adopt a theme that will reflect the season properly: **Tides of Winter**! As you might have guessed, in this iteration of the contest, we expect submissions all themed around anything winter-related. Whether it fills people with joy and everlasting Christmas spirit, or pulls out a different take on winter sports, anything is acceptable as long as it fits the theme.
```